fetch.bs

```diff
@@ -5074,21 +5073,14 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
  <li><p>Let <var>networkPartitionKey</var> be the result of
  <a for=request>determining the network partition key</a> given <var>request</var>.
 
- <li>
-  <p>Switch on <var>request</var>'s <a for=request>mode</a>:
-
-  <dl>
-   <dt>"<code>websocket</code>"
-   <dd><p>Let <var>connection</var> be the result of
-   <a lt="obtain a WebSocket connection">obtaining a WebSocket connection</a>, given
-   <var>request</var>'s <a for=request>current URL</a>.
+ <li><p>Let <var>dedicatedConnection</var> be true if <var>request</var>'s <a for=request>mode</a>
```

Comment:
This might change with WebSocket over HTTP/2?

Reply:
Yeah, so what should we do. Add a _dedicatedIfHTTP1_ parameter?

Reply:
It's not pretty, but it would work.